README.md

```diff
@@ -1,2 +1,21 @@
 # foreman_teamdynamix
-A Foreman Plugin for TeamDynamix
+A Foreman Plugin for TeamDynamix. It manages a host lifecycle as a corresponding Asset in TeamDynamix.
+
```

Comment:
We probably want an installation section also, like here: https://github.com/MiamiOH/foreman_custom_tab/blob/master/README.md#installation

Reply:
will fix this with issue#1 PR